ansible-tests/validations/controller-ulimits-check.yaml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: controller
+  vars:
+    metadata:
+      name: Check controller ulimits
+      description: >
+        This will check the ulimits of each controller.
+      groups:
+      - post-deployment
+    nofiles_default: 1025
+    nproc_default: 1025
+  tasks:
+  - name: Get nofiles limit
+    become: true
+    command: sh -c "ulimit -n"
+    register: nofilesval
+    changed_when: False
+  - name: Check nofiles limit
+    fail:
+      msg: >
+        nofiles is set to {{ nofilesval.stdout }}.  It should be at least 2048 or higher, depending on available resources.
```

Comment:
Not as designed.  I'm using "nofiles_default" as the set minimum to check the current value against.  Right now we are checking the determined value against a static value of 1025.